app/controllers/sprangular/orders_controller.rb

```diff
@@ -13,9 +13,13 @@ def index
   end
 
   def show
-    authorize! :show, @user
-
-    @order = Spree::Order.where(number: params[:id]).first!
+    token = request.headers['X-Spree-Order-Token']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.